tests/array/last.js

```diff
@@ -0,0 +1,12 @@
+var chai = require('chai');
+var expect = chai.expect;
+
+require('../../array/last');
+
+describe('last', function() {
+  it('should grab the last item from the array', function() {
+    var ary = ['t', 'e', 's', 't', 'tests'];
+
+    expect(ary.last).to.eql(["tests"]);
```

Comment:
If we're allowing you to specify first(x), last should be the same. @rampage1213 @zackify